plugins/org.eclipse.n4js/src/org/eclipse/n4js/validation/N4JSResourceValidator.java

```diff
@@ -41,6 +46,37 @@
 	@Inject
 	private OperationCanceledManager operationCanceledManager;
 
+	@Override
+	public List<Issue> validate(Resource resource, CheckMode mode, CancelIndicator cancelIndicator) {
+		if (resource instanceof N4JSResource) {
+			final N4JSResource resourceCasted = (N4JSResource) resource;
+			try {
+				resourceCasted.performPostProcessing(cancelIndicator);
+			} catch (Throwable th) {
+				// ignore this exception/error (we will create an issue for it below)
+			}
+			if (resourceCasted.isFullyProcessed()
+					&& resourceCasted.getPostProcessingThrowable() != null) {
+				// When getting here, we have an attempt to validate a resource that was post-processed but the
+				// post-processing failed (i.e. postProcessingThrowable!=null). Validating such a resource will usually
+				// cause a multitude of follow-up exceptions in many @Check methods (could easily be hundreds). Since,
+				// EMF/Xtext has the behavior of not aborting the overall validation due to an exception in a single
+				// @Check method, this will lead to many exceptions, which are all just follow-up issues of the problem
+				// that caused the post-processing to fail.
+				// 1) Since this is annoying and misleading, we just ignore all ordinary validation.
+				// 2) To not overlook the problem, we create a single validation issue (error) pointing out the problem.
+				// (for a test, see class AvoidFollowUpExceptionsInValidationTest)
```

Comment:
link to `AvoidFollowUpExceptionsInValidationTest `?

Reply:
in plain comments, links are not possible, afaik